dev/breeze/src/airflow_breeze/global_constants.py

```diff
@@ -221,6 +221,44 @@ def get_default_platform_machine() -> str:
 CURRENT_MSSQL_VERSIONS = ["2017-latest", "2019-latest"]
 DEFAULT_MSSQL_VERSION = CURRENT_MSSQL_VERSIONS[0]
 
+
+AIRFLOW_PYTHON_COMPATIBILITY_MATRIX = {
```

Comment:
I think we need to update https://github.com/apache/airflow/blob/main/dev/README_RELEASE_AIRFLOW.md by adding a new step to keep this dictionary updated after each release.

Reply:
Yeah. But I think we should add it as next step. We need to update a few steps in the release process after we complete all the SBOM generation and verify it:

* updating relased airflow version here (that should be likely even via pre-commit when new version is updated)
* regenerating SBOMS for Airflow
* regenerating SBOMS for images
* regenerating package metadata when providers are released
* regenerating SBOM for providers when released.

I think adding it now is a bit premature until we test it all :) - it might cause more harm and friction in the release process

Reply:
(BTW. Just testing this one :)

Reply:
I agree, we need to update, will do in following step once the process is well tested.